sequencing/vcf_qc/dxapp.json

```diff
@@ -102,62 +172,39 @@
       "label": "Additional Hard Filters",
       "class": "string",
       "optional": true
-    },
-    {
-      "group": "Build Options",
-      "name": "build_version",
-      "label": "NCBI Build Version",
-      "class": "string",
-      "choices": ["rgc_b38","gatk_b38","b37_decoy","hg38chr.fa"],
-      "default": "rgc_b38",
-      "optional": false
     }
   ],
   "outputSpec": [
     {
       "name": "vcf_out",
       "label": "Filtered VCF file",
       "class": "array:file",
-      "patterns": [
-        "*.vcf.gz"
-      ]
+      "patterns": ["*.vcf.gz"]
     },
     {
       "name": "vcfidx_out",
       "label": "Filtered VCF Index file",
       "class": "array:file",
-      "patterns": [
-        "*.tbi"
-      ]
+      "patterns": ["*.tbi", "*.vcf.gz.tbi"]
     }
   ],
   "runSpec": {
-  "execDepends": [
-      {
-        "name": "pigz"
-      },
-      {
-        "name": "tabix"
-      }
-    ],
-    "systemRequirements": {
-      "*": {
-        "instanceType": "mem1_ssd2_x8"
-      }
-    },
     "distribution": "Ubuntu",
     "release": "14.04",
+    "restartableEntryPoints": "all", 
     "interpreter": "bash",
-    "file": "src/vcf_qc.sh"
+    "file": "src/vcf_qc.sh",
+    "timeoutPolicy": {"*": { "hours": 120 }}
```

Comment:
Maybe we can reduce this to 2 or 3 days instead of 5 days?

Reply:
Thanks Naina.